dev-embedded/ponyprog/files/ponyprog-3.1.4_p20250501-usbwatcher-qt6.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/SrcPony/usbwatcher.h b/SrcPony/usbwatcher.h
```

Comment:
Please always add some explanation of patches at the top, as well as any links to upstream commits/PRs/bugs. (Apply this to all patches.)